server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -231,4 +243,24 @@ private String convertToString(ArrayList<String> comments) {
 
     return res.toString();
   }
+
+  /**
+   * Filters out a comment string so that it's readable and doesn't break the frontend.
+   *
+   * @param comment The comment to be filtered.
+   * @return A properly formatted comment
+   */
+  private String filterComment(String comment) {
+    String filteredComment = comment;
+
+    // Handles non-ASCII characters, newlines,
```

Comment:
This comment should be updated now that you're not filtering out all non-ASCII characters